Language/Classes/Instance_Methods/Method_noSuchMethod/forwarder_A03_t01.dart

```diff
@@ -23,53 +23,44 @@
 /// For a concrete class C, a noSuchMethod forwarder is implicitly induced for
 /// each member signature which is noSuchMethod forwarded.
 ///
-/// @description Checks that it is a compile-time error if a class has an
-/// interface member `m` and doesn't have a valid concrete implementation of `m`
-/// even if it has a non-trivial `noSuchMethod`
+/// @description Checks that 'noSuchMethod' forwarder forced by privacy always
+/// throws
 /// @author sgrekhov22@gmail.com
 
-abstract mixin class A {
-  String m([String s = ""]);
-}
-
-mixin class B {
-  String m() => "B";
-}
+import "no_such_method_lib.dart";
+import "../../../../Utils/expect.dart";
 
-class C1 extends B implements A {
-//    ^^
-// [analyzer] unspecified
-// [cfe] unspecified
-  dynamic noSuchMethod(Invocation i) => "C1";
+class C extends D0 {
+  String _m();
```

Comment:
I think the reason why we might want to have a private method named `_m`-in-this-library is that it must be a different method than the one named `_m`-in-no_such_method_lib.dart in `D0`, and we can detect this because they behave differently.

But it would still be helpful to have a comment to that effect. Perhaps something like

  // This `_m` is different from `D0._m`: They behave differently.